pytest.ini

```diff
@@ -1,2 +1,4 @@
 [pytest]
 junit_logging = all
+markers =
+	dummy: marks dummy tests to be executed only by different tests
```

Comment:
`... only by robottelo internal tests`  ?

Reply:
sorry, this is an outdated PoC. This got completely removed